src/views/home/UserInput.js

```diff
@@ -51,6 +92,10 @@ export default class UserInput extends Component {
         screenshotProp.height = parseInt(e.target.value) || '';
       }
       this.setState({ screenshotProp });
+    } else if (value != null) {
+      screenshotProp.userAgent = value || '';
+      this.setState({ screenshotProp });
+      debugger;
```

Comment:
Hmm... we definitely shouldn't have `debugger` directives making it into `master`